v0/destinations/singular/data/SINGULAREventNotificationConfig.json

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "destKey": "n",
+    "sourceKeys": "event",
+    "required": true
+  },
+  {
+    "destKey": "p",
+    "sourceKeys": "context.os.name",
+    "required": true
+  },
+  {
+    "destKey": "i",
+    "sourceKeys": "context.app.namespace",
+    "required": true
+  },
+  {
+    "destKey": "ip",
+    "sourceKeys": ["context.ip", "request_ip"],
+    "required": true
+  },
+  {
+    "destKey": "ve",
+    "sourceKeys": "context.os.version",
+    "required": true
+  },
+  {
+    "destKey": "att_authorization_status",
+    "sourceKeys": "context.device.adTrackingEnabled",
+    "required": true
+  },
+  {
+    "destKey": "custom_user_id",
+    "sourceKeys": "userId",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "utimȩ",
+    "sourceKeys": "timestamp",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "skan_conversion_value",
+    "sourceKeys": "properties.skan_conversion_value",
+    "required": false
+  },
+  {
+    "destKey": "skan_first_call_timestamp",
+    "sourceKeys": "properties.skan_first_call_timestamp",
+    "required": false
+  },
+  {
+    "destKey": "skan_last_call_timestamp",
+    "sourceKeys": "properties.skan_last_call_timestamp",
+    "required": false
+  },
+  {
+    "destKey": "e",
+    "sourceKeys": "properties.eventAttributes",
+    "required": false
+  },
+  {
+    "destKey": "is_revenue_event",
+    "sourceKeys": "properties.is_revenue_event",
```

Comment:
We can also do a check if this field is not presnet in the payload then we can check the presence of revenue key in the properties